lib/attr_encrypted/adapters/active_record.rb

```diff
@@ -54,15 +54,15 @@ def attr_encrypted(*attrs)
             options = attrs.extract_options!
             attr = attrs.pop
             attribute attr if ::ActiveRecord::VERSION::STRING >= "5.1.0"
-            options.merge! encrypted_attributes[attr]
+            options.merge! legacy_encrypted_attributes[attr]
 
             define_method("#{attr}_was") do
```

Comment:
this method never worked and is already broken in the non forked version. I did not change it as we don't use it for encrypted attributes at the moment.